src/api/docker-files/Dockerfile.frontend-backend

```diff
@@ -0,0 +1,13 @@
+# layer on top to include backend for minitest
```

Comment:
I don't like this. Why aren't we using the [CI locally](https://circleci.com/docs/2.0/local-cli/) to run the tests?

Reply:
because it doesn't run emacs?

Reply:
Ah... 

`Note: The Local CLI does not yet support Workflows.`

Reply:
circleci CLI doesnt work out of the box at least. Even without workflow, it should be possible to run a circleci build minitest but it doesn't. Maybe in a near future..

Reply:
because our checkout comes from the cache - generated in the checkout_code test. And caches aren't supported by local CLI either